libs/dh/metering-point/feature-overview/src/lib/dh-meter-data.component.ts

```diff
@@ -68,46 +62,84 @@ type DhMeterData = {
     }
   `,
   template: `
-    <div class="page-grid" *transloco="let t; read: 'meteringPoint.meterData'">
-      <watt-datepicker [formControl]="date" [max]="maxDate" [label]="t('dateLabel')" />
+    <watt-data-table
+      vater
+      inset="ml"
+      [enableSearch]="false"
+      [error]="dataSource.error"
+      [ready]="dataSource.called"
+      *transloco="let t; read: 'meteringPoint.meterData'"
+    >
+      <watt-data-filters>
+        <vater-stack align="flex-start">
+          <watt-datepicker [formControl]="date" [max]="maxDate" [label]="t('dateLabel')" />
+        </vater-stack>
+      </watt-data-filters>
+      <watt-table
+        *transloco="let resolveHeader; read: 'meteringPoint.meterData.table.columns'"
+        [resolveHeader]="resolveHeader"
+        [columns]="columns"
+        [dataSource]="dataSource"
+        [loading]="dataSource.loading"
+        sortDirection="desc"
+        [sortClear]="false"
+      >
+        <ng-container
+          *wattTableCell="columns.observationTime; header: t('columns.time'); let element"
+        >
+          {{ element.observationTime | wattDate: 'long' }}
+        </ng-container>
 
-      <watt-card vater fill="vertical">
-        <vater-flex fill="vertical" gap="m">
-          <h3>{{ date.value | wattDate | dhEmDashFallback }}</h3>
+        <ng-container *wattTableCell="columns.quantity; header: t('columns.value'); let element">
+          {{ element.quantity }}
+        </ng-container>
```

Comment:
suggestion: the table uses `resolveHeader` but every column also has `header` property. Use only one of them.

Reply:
Aha, to much copy / paste :-)